integration/fuse/src/main/java/alluxio/fuse/AlluxioFuseUtils.java

```diff
@@ -359,6 +364,36 @@ public static boolean isFuseInstalled() {
     return false;
   }
 
+  /**
+   * @return true if os version is matched to use macFuse
+   */
+  public static boolean isMacFuse() {
```

Comment:
this just means the version is supported by alluxio, not actually means the mac or not?

Reply:
This has nothing to do with alluxio, but to determine the version of fuse supported by different versions of mac os.